algorithms/CPlusPlus/Linked-Lists/reverseInGroupsOfK.cpp

```diff
@@ -0,0 +1,34 @@
+// Description := function of reverse a linkedlist in groups of size K .
+
+// Time and space complexity :-
+// Time Complexity = O(N) and Space Complexity = O(N)
```

Comment:
If a LinkedList is there, then the space complexity would be O(1)

Reply:
No, recursion stack space is there bro.